### PR TITLE
feat:contributing guide is added

### PR DESCRIPTION
**Pull request by Kshitiz-Mhto**
## Related Issue

Closes: #5

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] I have followed the code style of the project
- [x] I have tested my code, and it works without errors

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/CeruleanCodersComm/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/krshname/username-checker/pull/18"></a>

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
@frazie wanted to work on this..

**Comment by KendallDoesCoding** [comment]
Hey Kshitz,

No worries. I do the same thing half the time, but just letting you know
that if its assigned, you should ask before you work on the issue, because
incase the other person has started work it wastes everyones time.

Frazie doesnt seem to have started work on this.

Nono, no worries. The code looks good, don't close the PR :))

Thanks,

File: .github/CONTRIBUTING_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Bugs are tracked as github [issues](https://github.com/krshname/username-checker/issues). After you've determined which repository your bug is related to, create an issue on that repository.
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. 
+
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **Include screenshots or animated GIFs** which show you following the described steps and clearly demonstrate the problem. If you use the keyboard while following the steps, use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
```
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply .github/CONTRIBUTING_GUIDE.md:21]: You can delete the following lines as they are not required

```
If you use the keyboard while following the steps, use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
```

Comment (KendallDoesCoding) [inline .github/CONTRIBUTING_GUIDE.md:21]:
```suggestion
* **Include screenshots or animated GIFs** which show you following the described steps and clearly demonstrate the problem.
```

delete not required text

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by krshkodes — APPROVED** [review]
Thank you! Looks great!

**Pull request merged**